### DISCUSSION TITLE
Streamline thickness in velocity velocity_embedding_stream

### DISCUSSION THREAD

**Discussion by torgheh** [theislab/scvelo#905] [Q&A] [answered]
Hello and thanks for you amazing software and support.

You can see in the `velocity_embedding_stream` plot that line thicknesses are different. I could not find an explanation for this difference and what does it correspond to neither in the paper nor in the docs. Does the thickness correlate with average velocity of the neighboring cells?

![image](https://user-images.githubusercontent.com/14017891/180211558-029c5520-e49b-47b2-8bb9-441d746d3c71.png)

**Comment by WeilerP** [comment, marked as answer]
The line thickness is related to the magnitude of the velocity.
